apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-index.presenter.tsx

```diff
@@ -18,14 +19,20 @@ export const VocabularyListIndexPresenter = ({
         name: ({ name }) => renderMultilingualTextCell(name),
     };
 
+    const matchers: Matchers<IVocabularyListViewModel> = {
```

Comment:
Do we have a Cypress e2e test for this flow? We should probably test the filtering behaviour. We could do this by stubbing the API response (may already be done) and returning dummy data and then writing a UX test where we fill out the search bar with various search strings designed to match or not match one of the items.